backend/src/models/answer.js

```diff
@@ -37,6 +26,16 @@ class Answer {
     }
 }
 
-module.exports = (function(db) {
+let Singleton = null
+
+/**
+ * @param {object} db defaults to the db instance
+ */
+module.exports = function(db) {
```

Comment:
this seems like a reasonable way to do it; defaulting to the actual database

Reply:
👍